src/styles/stopwatch.scss

```diff
@@ -0,0 +1,59 @@
+.stopwatch {
+  position: relative;
+  width: 80vmin;
+  height: 80vmin;
+  border: $border-width dotted $border-color;
+  border-radius: 50%;
+
+  &__minutes-hand {
+    position: absolute;
+    height: $minutes-hand-height;
+    width: $hand-width-minutes;
+    background-color: $minutes-hand-color;
+    bottom: 50%;
+    left: calc(50% - (#{$hand-width-minutes} / 2));
+    transform-origin: bottom center;
+    animation: rotate 3600s steps(60) infinite;
+  }
+
+  &__seconds-hand {
+    position: absolute;
+    height: $seconds-hand-height;
+    width: $hand-width-seconds;
+    background-color: $seconds-hand-color;
+    bottom: 50%;
+    left: calc(50% - (#{$hand-width-seconds} / 2));
+    transform-origin: bottom center;
+    animation: rotate 60s linear infinite;
```

Comment:
As with the minutes hand, consider breaking the `animation` shorthand into individual properties for the seconds hand. This will improve the readability and maintainability of your code.